go/vt/vtexplain/vtexplain_vttablet.go

```diff
@@ -75,31 +77,147 @@ func newFakeTablet() *fakeTablet {
 		dbconfigs.AppConfig, // These tests only use the app pool.
 	)
 
-	target := querypb.Target{TabletType: topodatapb.TabletType_MASTER}
+	target := querypb.Target{
+		Keyspace:   t.Keyspace,
+		Shard:      t.Shard,
+		TabletType: topodatapb.TabletType_MASTER,
+	}
 	tsv.StartService(target, dbcfgs, mysqld)
 
 	// clear all the schema initialization queries out of the tablet
 	// to avoid clutttering the output
-	tablet.queries = nil
+	tablet.mysqlQueries = nil
 
 	return &tablet
 }
 
-// Execute hook called by SandboxConn as part of running the query.
-func (tablet *fakeTablet) Execute(ctx context.Context, target *querypb.Target, query string, bindVars map[string]*querypb.BindVariable, transactionID int64, options *querypb.ExecuteOptions) (*sqltypes.Result, error) {
-	logStats := tabletenv.NewLogStats(ctx, "FakeQueryExecutor")
+var _ queryservice.QueryService = (*fakeTablet)(nil) // compile-time interface check
 
-	plan, err := tablet.tsv.GetPlan(ctx, logStats, query)
-	if err != nil {
-		return nil, err
-	}
+// Begin is part of the QueryService interface.
+func (tablet *fakeTablet) Begin(ctx context.Context, target *querypb.Target, options *querypb.ExecuteOptions) (int64, error) {
```

Comment:
nit: Receivers must be 1 or 2 characters only: https://github.com/golang/go/wiki/CodeReviewComments#receiver-names

I would just go here with `t (or maybe `ft`).

Reply:
Fixed. Thanks.